CloudVision_ChangeControlManager_Scripts/Migrate_to_CVaaS.py

```diff
@@ -0,0 +1,28 @@
+import json
+from cvplibrary.auditlogger import alog
+from cvplibrary import Device, CVPGlobalVariables, GlobalVariableNames
+ip = CVPGlobalVariables.getValue(GlobalVariableNames.CVP_IP)
+scriptArgs = CVPGlobalVariables.getValue(GlobalVariableNames.SCRIPT_ARGS)
+
+# This script can be used to migrate nodes from CVP on-prem to CVaaS
+# An onboarding token needs to be generated on CVaaS and defined in the argument in the YAML file
+
+target = Device(ip)
+TOKEN = scriptArgs["Token"]
+VRF = scriptArgs["VRF"]
+
+# Write an entry to CVP Log
+alog("Migrating node to CVaaS")
+
+# Migration
+try:
+    target.runCmds(["enable", \
+     "bash timeout 2 echo '%s' > /tmp/cv-onboarding-token" % (TOKEN), \
```

Comment:
there's a better way to do this using eAPI, can you try this:
`{"cmd": "copy terminal: file:/tmp/cv-onboarding-token", "input": TOKEN}`

Reply:
Thanks